tests/test_existing_excerpt.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+import time
+import requests
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.keys import Keys
+
+@pytest.fixture
+def prerequisite(base_url, selenium):
+    # create new excerpt only for this testing
+    selenium.get('{0}/admin/login/'.format(base_url))
```

Comment:
Please extract the login functionality into a separate fixture, and move that code into a new file called `conftest.py` inside the `tests/` directory.
The fixtures inside that file will be loaded automatically (pytest magic). You then can extend this fixture and remove the login code:
```python
@pytest.fixture
def prerequisite(login, base_url, selenium):
....
```